techmatters/flex-plugins
# CHI-1644: Tidy up

## Description

* Add basic error handling to view resource page
* Align service configuration & URLs with back end

### Checklist

### Related Issues
CHI-1644

### Verification steps

* Ensure a version of HRM with resources is deployed without a resource with id 'EXAMPLE_RESID' in it's DB
* Load the resources page, should show a 'Not Found' error
* Add the resource with that ID
* Resources page should show the name provided to that resource in the DB
